.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macOS-latest, windows-latest ]
+        python-version: [ '3.8' ]
+        include:
+          - os: ubuntu-latest
+            python-version: '3.9'
+            tasks: tests
+          - os: ubuntu-latest
+            python-version: '3.10'
+            tasks: tests
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Ubuntu - install libsndfile
+      run: |
+        sudo apt-get update
+        sudo apt-get install --no-install-recommends --yes libsndfile1
+      if: matrix.os == 'ubuntu-latest'
+
+    - name: Install package
```

Comment:
```suggestion
    - name: Install packages
```
OR
```suggestion
    - name: Install required libraries
```

Reply:
With "package" I refer to the actual Python package that we want to test (`audbcards`). Dependencies will be of cause installed as well.

If we change to "Install required libraries" it could also mean that we only install dependencies, but not the Python package of this repo.

Reply:
Ah okay I misunderstood I thought we don't install the actual package only the dependencies in `setup.cfg`.

Reply:
No, we also install the package when running `pip install -r requirements.txt`, compare https://setuptools.pypa.io/en/latest/userguide/development_mode.html